Software/res/moodlamps/static.mjs

```diff
@@ -0,0 +1,9 @@
+export const name = "Static (default)"
+
+// note: in static or very slow animations it's better to keep the refresh interval under 500ms to avoid bugs
```

Comment:
250 is also under 500. Did you have some bugs because of this? At some point I added `FAKE_GRAB_INTERVAL` because some devices turn off after no input in a second.

Reply:
it's been a while, I think the issue was that putting a too high of a value was causing lag when changing modes, so the lower the better
and probably also with "dont send data unless changed"
I'll double check

Reply:
"putting a too high of a value was causing lag when changing modes" - switching the mode should reset the timer, so if it does cause a delay, it would be worth checking out why.

Reply:
I'm only 99% sure about this, but I found and fixed the lag that occurred on lamp change: there was no call to update so you had to wait for the timer to fire